### PR TITLE
Contributors

### PR DESCRIPTION
**Pull request by gkellogg**
***

### :boom: Error: 500 Internal Server Error :boom: ###

[PR Preview](https://github.com/tobie/pr-preview#pr-preview) failed to build. _(Last tried on Feb 5, 2023, 11:38 PM UTC)_.

<details>
<summary>More</summary>

PR Preview relies on a number of web services to run. There seems to be an issue with the following one:

:rotating_light: [Spec Generator](https://www.w3.org/2015/labs/) - Spec Generator is the web service used to build specs that rely on ReSpec.

:link: [Related URL](https://labs.w3.org/spec-generator/?type=respec&url=https%3A%2F%2Fraw.githubusercontent.com%2Fw3c%2Frdf-xml%2F4a92f3b49b659523011b1c5a6cbaa032988d5826%2Fspec%2Findex.html%3FisPreview%3Dtrue)

```

😭  Sorry, there was an error generating the HTML. Please report this issue!
Specification: http://labs.w3.org/spec-generator/uploads/BtykMO/spec/index.html?isPreview=true%3FisPreview%3Dtrue&publishDate=2023-02-05
ReSpec version: 32.7.0
File a bug: https://github.com/w3c/respec/
Error: Error: Evaluation failed: Timeout: document.respec.ready didn't resolve in 28326ms.
    at ExecutionContext._ExecutionContext_evaluate (/u/spec-generator/node_modules/puppeteer-core/lib/cjs/puppeteer/common/ExecutionContext.js:229:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ExecutionContext.evaluate (/u/spec-generator/node_modules/puppeteer-core/lib/cjs/puppeteer/common/ExecutionContext.js:107:16)
    at async generateHTML (/u/spec-generator/node_modules/respec/tools/respecDocWriter.js:221:12)
    at async toHTML (/u/spec-generator/node_modules/respec/tools/respecDocWriter.js:92:18)
    at async Object.generate [as respec] (file:///u/spec-generator/generators/respec.js:15:44)
    at async file:///u/spec-generator/server.js:252:48

```

_If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please [file an issue](https://github.com/tobie/pr-preview/issues/new?title=Error%20not%20surfaced%20properly&body=See%20w3c/rdf-xml%234.)._
</details>

### CODE REVIEW

File: spec/index.html [hunk 1]
```diff
@@ -80,7 +80,7 @@
     section. Since the edits to this document do not invalidate
     previous implementations the Director decided no new implementation report was required. </p>
 
-  <section id="related" data-include="./common//related.html"></section>
+  <section id="related" data-include="./common/related.html"></section>
```
Comment (afs) [inline spec/index.html:83]: This does not work (for me at least) when viewed locally (all specs). I get a reSpec error and a reSpec warning.

Is there some magic I'm missing?

**Review by afs — APPROVED** [review]

**Comment by gkellogg** [comment]
There’s an outstanding bug in ReSpec (https://github.com/w3c/respec/issues/4378#issuecomment-1416860913) that halts processing in this spec, in particular. I have a workaround that re- defines the “SAX” reference in localBibliogrsph, but wanted to give them a chance to fix it. (If you just remove `[[SAX]]` in your local copy, it should build).

but, it seems to work in all other RDF specs for me. The problem I mentioned is only on RDF-XML, which has prevented merging this PR. The set of docs seems to show for [rdf-concepts](https://w3c.github.io/rdf-concepts/spec/#related). Where are you seeing the problem?

**Comment by gkellogg** [comment]
The data-include stuff might be a problem if you just load the file in a browser window directly. I use `python -m http.server` and access via localhost.

**Comment by afs** [comment]
Yes, I load the specs as `file:///`.

I can use Fuseki (!!).

**Pull request merged**